sdk-facebook/src/main/java/co/reachfive/identity/sdk/facebook/FacebookProvider.kt

```diff
@@ -67,42 +63,34 @@ class ConfiguredFacebookProvider(
     ) {
         LoginManager.getInstance()
             .registerCallback(callbackManager, object : FacebookCallback<LoginResult> {
-                override fun onSuccess(result: LoginResult?) {
-                    val accessToken = result?.accessToken?.token
-                    if (accessToken != null) {
-                        val loginProviderRequest = LoginProviderRequest(
-                            provider = name,
-                            providerToken = accessToken,
-                            clientId = sdkConfig.clientId,
-                            origin = origin,
-                            scope = scope.joinToString(" ")
+                override fun onSuccess(result: LoginResult) {
+                    val accessToken = result.accessToken.token
```

Comment:
Ce cas d'erreur ne se présente jamais ?

Reply:
Discussed: the returned values are no longer nullable ✔️